src/templates/InstantSearch.js/src/app.js.hbs

```diff
@@ -25,15 +25,15 @@ search.addWidget(
     {{#if attributesToDisplay}}
     templates: {
       item: `
-        <article>
-          <h1>\{{#helpers.highlight}}{ "attribute": "{{attributesToDisplay.[0]}}" }\{{/helpers.highlight}}</h1>
-          {{#each attributesToDisplay}}
-          {{#unless @first}}
-          <p>\{{#helpers.highlight}}{ "attribute": "{{this}}" }\{{/helpers.highlight}}</p>
-          {{/unless}}
-          {{/each}}
-        </article>
-      `,
+<article>
```

Comment:
Why did you choose to put this without indentation?

Reply:
That's how Prettier formats it and it's easier to edit it so I figured why not. No strong opinion though.